src/dbus_api/pool/pool_2_1/api.rs

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use dbus::tree::{Access, EmitsChangedSignal, Factory, MTFn, Property};
+
+use crate::dbus_api::{consts, pool::pool_2_1::props::get_pool_encrypted, types::TData};
+
+pub fn encrypted_property(f: &Factory<MTFn<TData>, TData>) -> Property<MTFn<TData>, TData> {
+    f.property::<bool, _>(consts::POOL_ENCRYPTED_PROP, ())
+        .access(Access::Read)
+        .emits_changed(EmitsChangedSignal::True)
```

Comment:
I'ld like to go with ```EmitChangedSignal::Const```, as opposed to ```True```. Otherwise, to make our code consistent we would have to make a change in ```EventHandler::notify``` to handle possible changes. And I don't think we really expect to be doing encryption of a non-encrypted pool any time soon. If we ever end up doing that, we'ld want to bump the D-Bus version number, but that will be a small effort compared to the other work we would have to do to make that happen.